middlewares/validatePostQuestion.js

```diff
@@ -1,19 +1,13 @@
 const validatePostQuestion = (req, res, next) => {
-    let { title, body } = req.body;
+    let { questionTitle, questionDescription } = req.body;
   
-    title = title && title.toString().trim();
-    body = body && body.toString().trim();
-    if (!title) {
+    questionTitle = questionTitle && questionTitle.toString().trim();
+    questionDescription = questionDescription && questionDescription.toString().trim();
+    if (!questionTitle && !questionDescription) {
```

Comment:
1. If I provide a `questionDescription` and I fail to provide a `questionTitle`, I would not get the error. Basically, this won't work in cases where either one of the title or description is not provided. You want to use the `||` operator instead of the `&&`. 
2. Since this checks for both title and description, the error message should not be `Title is required`, but something that tells the user both the title and description is required.

Reply:
on this now

Reply:
done